app/controllers/web-payments/handle-auth-response-controller.js

```diff
@@ -11,13 +11,15 @@ const Charge = require('../../models/charge')
 const { withAnalytics } = require('../../utils/analytics')
 const paths = require('../../paths')
 const { getSessionVariable, deleteSessionVariable } = require('../../utils/cookies')
+const State = require('../../../config/state')
 
 // constants
 const routeFor = (resource, chargeId) => paths.generateRoute(`card.${resource}`, { chargeId })
 const webPaymentsRouteFor = (resource, chargeId) => paths.generateRoute(`webPayments.${resource}`, { chargeId })
 
 const redirect = res => {
   return {
+    toAuth3dsRequired: (chargeId) => res.redirect(303, routeFor('auth3dsRequired', chargeId)),
```

Comment:
I've made this redirect to auth3dsRequired rather than authWaiting as authWaiting would cause two redirects in total.